plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtils.java

```diff
@@ -46,6 +47,21 @@ public static Dictionary constructAssetPackStateDictionary(
     return returnDict;
   }
 
+  public static Dictionary constructAssetPackStatesDictionary(
```

Comment:
I modified this function so that it can take a Dictionary constructed by stream. I also decided to keep appendToAssetPackStatesDictionary() since it is used as a helper function for testcases.